examples/axisartist/demo_axisline_style.py

```diff
@@ -1,3 +1,10 @@
+"""
+================
+Axis Line Styles
```

Comment:
Looks like most already converted examples only capitalize the first word (which I prefer too), so let's stay consistent (applies to this and all files below).